web/packages/build/package.json

```diff
@@ -60,28 +34,15 @@
     "eslint-plugin-react": "^7.34.3",
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-testing-library": "^6.2.2",
-    "events": "3.3.0",
-    "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
-    "jest-fail-on-console": "^3.3.0",
     "jest-styled-components": "^7.2.0",
     "jsdom": "^21.1.0",
-    "jsdom-testing-mocks": "^1.13.0",
-    "msw": "^0.47.4",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "react-is": "^18.3.1",
-    "react-select-event": "^5.5.1",
-    "react-transition-group": "^4.4.5",
     "rollup-plugin-visualizer": "^5.12.0",
-    "styled-components": "5.3.11",
     "typescript-eslint": "^7.14.1",
-    "vite": "^5.3.2",
     "vite-plugin-wasm": "^3.3.0",
     "vite-tsconfig-paths": "^4.3.2"
   },
   "devDependencies": {
-    "msw-storybook-addon": "^1.8.0",
     "ts-loader": "^9.5.1"
```

Comment:
`ts-loader` is a Webpack leftover I think, no?

Reply:
It's still used by storybook https://github.com/gravitational/teleport/blob/089c32094f4773638b5c2183bbeeeb21dd548d39/web/.storybook/main.js#L100